solar-land/scss/_typography.scss

```diff
@@ -0,0 +1,35 @@
+// Typography for Quicksand font
+.quicksand-light {
+  font-family: "Quicksand", sans-serif;
+  font-weight: 300;
+  font-optical-sizing: auto;
+  font-style: normal;
+}
+
+.quicksand-regular {
+  font-family: "Quicksand", sans-serif;
```

Comment:
You repeated the font family in all of this file, it means it should be manageable by using variable